en/operations/configuration-server.html

```diff
@@ -72,9 +72,8 @@ <h2 id="redundancy">Redundancy</h2>
 <p>
   When there are multiple config servers, the <a href="config-proxy.html">config proxy</a>
   will pick a config server randomly (to achieve load balancing between config servers).
-  The config proxy is fault-tolerant and will switch to another config server if the one it is using becomes unavailable
-  or there is an error in the configuration it receives.
-  With only one config server configured, it will continue using that in case of errors.
+  The config proxy is fault-tolerant and will switch to another config server (if there is more than one)
```

Comment:
There are always 3 config server, so suggesting there could be one is confusing?